gnovm/pkg/gnolang/machine.go

```diff
@@ -881,10 +888,17 @@ const (
 	OpReturnCallDefers  Op = 0xD7 // TODO rename?
 )
 
+const GasFactorCpu int64 = 1
```

Comment:
Good catch, It should be int64, I removed the GasFactorAlloc since it is no longer needed.

Reply:
There should be a single place where it is documented all the constants, and there it should also be written that the gasFactorCPU must be a positive integer; otherwise people might modify gas constants/parameters in the future and then get stuck here, because they thought gasFactorCPU could be a fraction. 

If the intent is for "gas" to have the same unit of measurement as a CPU clock cycle, then that should be documented as the design philosophy; and that if we want more granularity (since 1 GNO CPU > 1 bare-metal CPU cycle) we can consider a higher factor than 1. Makes sense to me, but should be documented as such.